### ISSUE TITLE
Emphasize *.pov files

### ISSUE THREAD

**Issue by wolispace** [vpinball/vpinball#98]
Please add a dedicated on-screen button or a keyboard shortcut to save a *.pov file.

Currently its hidden behind File / Export / POV.

This is a fiddly thing to do while setting up a cabinet and also a bit hidden from new users who are not familiar with the benefits of *.pov files.

This could be a button in the grid next to the Camera (F6) and Play (F5) buttons .. maybe assign F4 or F7 to keep it in sequence an easy to remember.
 
Also find a way to highlight the concept of AutoPOV.pov.   Maybe File / Export / AutoPOV.pov.

**Comment by sverrewl** [comment]
I don't think it is a good idea to "expose" autopov.pov in that way. It is too powerful and will create a lot of forum posts from people who are not aware of what it can do. People who are really interested of tweaking their cabs; should read documentation and/or ask in the forums. The casual user, getting all his table all of a sudden, "broken", is not what we need.

**Comment by wolispace** [comment]
Fair enough re emphasising autopov.

Can we atleast have a keyboard shortcut to export pov file for current table?

When working in a cabinet getting to menu selected each time the camera position is adjusted is a fiddly pain!

It doesn't even have to be publicised, like autopov.  So just making shift F7 or whatever export the current pov file.

I guess I could use AutoIt3 but if it's built into vpx it's one less thing to configure as setup.

**Comment by sverrewl** [comment]
Understand the reason behind the request. Being a guy that has done POV's for personal use hundreds of times, change tables scripts close to 1k, I understand the frustration of efficiently want to change some value, being pov or whatever .. With all these year on my back though. Almost all my attempts of doing something "globally", has failed. There is machines that have the wrong dimensions in the first place. If I where to try to ... lets say - make a global pov for all my tables, I would end in a pile of hurt. Personally, I kind of collect, at the same time as I'm playing. I try to get every game to get as close to the real deal as possible. Ohter in this community, want to fill their sceen, to the max, not caring if the round figures on the pf looks like eggs, if they can fit their whole screen. So, yeah .. I understand, but, well, at some point. You should just accept that not everyone wants yous preferences, and making a button dedicated for wolispace might not be a valid request :)

**Comment by sverrewl** [comment]
Really ? Is it needed ? On many tables I do go along with what the author has suggested, other times, I want to change it. So, that mean I would go into the setup, put in my often used values, go into each value I want to change, then save. It is not a 1 button task anyway, some guys release "FS" without the "normal" 270. I don't see how this feature your requesting is very time saving in the end. Sure, if you decide to take the time to "not agree" with the author, you'll most probably spend some minutes or maybe more, to get it to where you like it. How much time does it really ave you then, needing to got to file -> save -> pov ? instead of F7 ? Very little. Coming for a guy that has change, surely 100's of tables, and not to speak for 1k+ of table scripts. I know how frustraing it cab be, sitting behind a windows gui to get things done - believe me.

**Comment by wolispace** [comment]
I think my original request has been lost in autopov confusion.

All I would like is a keypress like F7, or similar, that will simply do File / Export / POV.

We have standard shortcuts like Ctrl+S = File / Save, and custom shortcuts like F5 = Table / Play  and F6 = Table / Run

So adding a new shortcut to just save current pov would be a great time saver.. for me at least .

I find it much more efficient to press a key than hunt down three-menu-deep using a mouse on a 90 degree rotated display when the menus are really wide de to long file paths and names!

**Comment by spoyser** [comment]
Like most windows applications you could just use Alt-F to open the file menu, then P to open the Export Menu item, then P to select the POV item.
So, Alt-F P P
Given then it's two Ps its surprisingly quick

**Comment by francisdb** [comment]
`F12` - `1` (start) on 10.8.0

**Comment by jaoxford** [comment]
> Like most windows applications you could just use Alt-F to open the **F**ile menu, then P to open the Ex**p**ort Menu item, then P to select the **P**OV item. So, Alt-F P P Given that it's two Ps its surprisingly quick

That's the quickest way I found of accessing menu items.
Is there any way that user's can create their own shortcuts? I think that could be a cool feature.
